### PR TITLE
Bluemaxima.Voyager version 18.05

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110297&drop=dogfoodAlpha

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
That website seem downloading very slow.

**Comment by BrandonWanHuanSheng** [comment]
It look like it open up 7-zip console.

**Comment by BrandonWanHuanSheng** [comment]
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by upintheairsheep** [comment]
How do we deal with self extracting ZIP files with no shortcuts?

File: manifests/b/Bluemaxima/Voyager/18.05/Bluemaxima.Voyager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Bluemaxima.Voyager
+PackageVersion: 18.05
+Installers:
+- InstallerUrl: https://bluemaxima.org/voyager/Voyager.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: 8393EDFF65479BC2FF1828B2AED9170540D8517C86AECA522E13DFC83573224D
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
```
Comment (stephengillie) [inline manifests/b/Bluemaxima/Voyager/18.05/Bluemaxima.Voyager.installer.yaml:13]:
```suggestion
    Silent: -gm2
    SilentWithProgress: -y
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**